CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ Most recent version numbers *should* follow the [Semantic Versioning](https://se
 ### Added
 - option to choose different models and parameters for B1-correction of MTsat
 - set default WM percent value in hmri_defaults.
+- fix crash if input images have different matrix sizes, and warn
```

Comment:
@Barisevrenugur: This should have been put in the "Fixed" subsection.

Reply:
As it's just the CHANGELOG (and not code), I think you can just fix it directly on the master branch.